packages/traceability-schemas/scripts/build-credentials.js

```diff
@@ -38,11 +39,31 @@ const issueCreds = async (credTemplate, schemaName) => {
       }),
       documentLoader,
     });
-    const result = await vcjs.ld.verifyCredential({
+
+    // check if VC has credential status
+    let loopvar = credstatus.included.length;
+    let remove = false;
+      while (loopvar > 0) {
+        if (credstatus.included[loopvar - 1] === schemaName) {
+          remove = true;
+        }
+        loopvar -= 1;
+      }
+      let result = {};
+    if (!remove) {
+      result = await vcjs.ld.verifyCredential({
       credential: verifiableCredential,
       suite: new Ed25519Signature2018(),
       documentLoader,
     });
+      } else {
+          result = await vcjs.ld.verifyCredential({
+          credential: verifiableCredential,
+          suite: new Ed25519Signature2018(),
+          documentLoader,
+          checkStatus: async () => ({ verified: true })
```

Comment:
although to be clear, you are not checking the status, so this might allow broken example to slip into the spec.

Reply:
@OR13 ... you are right if this were real stuff then in the async function, you would check the status of the revocation credential, etc. Since this is a toy example and the links etc are all made up we do not have to worry about the case of a broken link. This simply says that for this example we assume the VC has not been revoked.

Reply:
https://github.com/w3c-ccg/traceability-vocab/issues/183